src/CodeGenerator/src/Generator/RequestSerializer/RestJsonSerializer.php

```diff
@@ -98,7 +98,7 @@ public function generateRequestBuilder(StructureShape $shape): array
 
     protected function dumpArrayBoolean(string $output, string $input, Shape $shape): string
     {
-        return strtr('$payloadOUTPUT = INPUT ? "true" : "false";', [
+        return strtr('$payloadOUTPUT = (bool) INPUT;', [
```

Comment:
That were wrong (and nevere used)

Reply:
Could you revert this please?

Reply:
why? I didn't find any documentation about this behavior (nor in AWS SDK code)

Reply:
Oh, We use boolean like this on other places. 

my bad.